squire_sdk/src/server/gathering/hall.rs

```diff
@@ -64,18 +65,23 @@ pub enum GatheringHallMessage {
 /// through message passing and tokio tasks.
 #[derive(Debug)]
 pub struct GatheringHall<P: ActorState<Message = PersistMessage>> {
```

Comment:
Looking at this again and thinking about it, it seems like we can just remove this type restriction. (That it requires a phantom should have been a sign). I'm going to push a change that removes it, lmk what you think.

You added it here:

f5a81892864c1ebdb3b9f397501373d9e9cc51ff